src/pages/single-product.html

```diff
@@ -1,17 +1,387 @@
+                        <img src="../assets/images/pages/single-product/product-image.png" alt="">
+                    </li>
+                    <li>
+                        <img src="../assets/images/pages/single-product/product-image.png" alt="">
+                    </li>
+                    <li>
+                        <img src="../assets/images/pages/single-product/product-image.png" alt="">
+                    </li>
+                    <li>
+                        <img src="../assets/images/pages/single-product/product-image.png" alt="">
+                    </li>
+                </ul>
+                <!-- ./image gallery -->
+
+                <hr class="border-[#BFC4D580] my-[42px]">
+
+                <!-- details -->
+                <div class="flex flex-col basis-3/4 justify-start">
+                    
+                    <!-- title -->
+                    <div class="flex justify-start items-center">
+                        <img src="../assets/images/icons/box.png" alt="Notebook Icon" class="w-[32px] h-[32px]">
+                        <h2 class="pr-2 text-[24px]">ویژگی‌های محصول</h2>
+                    </div>
+                    <!-- ./title -->
+
+                    <!-- attributes -->
+                    <ul class="list-disc list-inside mt-[30px]">
+                        <li class="flex justify-start border-b py-[26px] pt-0">
+                            <div class="basis-1/4 text-gray-500">▪ رنگ</div>
+                            <div>صورتی، سبز، بنفش، آبی</div>
+                        </li>
+                        <li class="flex justify-start border-b py-[26px]">
+                            <div class="basis-1/4 text-gray-500">▪ سایز</div>
+                            <div>20×25, 13×18, 10×15</div>
+                        </li>
+                        <li class="flex justify-start py-[26px] pb-0">
+                            <div class="basis-1/4 text-gray-500">▪ جنس قاب</div>
+                            <div>MDF</div>
+                        </li>
+                    </ul>
+                    <!-- ./attributes -->
+
+                </div>
+                <!-- ./details -->
+
+            </div>
+            <!-- ./description -->
+
+        </div>
+        <!-- ./description box -->
+
+        <!-- seperator -->
+        <div class="h-2 bg-[#E5E8EF] mt-[42px] -mx-8"></div>
+        <!-- ./seperator -->
+
+        <!-- carousel -->
+        <!-- head -->
+        <div class="flex justify-between items-center my-[57px]">
+            
+            <!-- title -->
+            <h1 class="text-[32px]">
+                محصولات مشابه
+            </h1>
+            <!-- ./title -->
+
+            <!-- action buttons -->
+            <div class="flex gap-2">
+                <div class="flex justify-center items-center h-[60px] w-[60px] bg-[#F5F8FC]">
+                    <img src="../assets/images/icons/arrow-right-line.svg" alt="Arrow Icon" class="opacity-60 rotate-180">
+                </div>
```

Comment:
It's clickable elements. So, It should be a button or defined as a button via `role=button`.